pp/go/cppbridge/prometheus_relabeler.go

```diff
@@ -525,17 +527,29 @@ func NewStaleNansState() *StaleNansState {
 	return s
 }
 
-func (s *StaleNansState) Reset() {
-	prometheusRelabelStaleNansStateReset(s.state)
-}
-
 // RelabelerStats statistics return from relabeler.
 type RelabelerStats struct {
 	SamplesAdded uint32
 	SeriesAdded  uint32
 	SeriesDrop   uint32
 }
 
+// Add another stats.
+func (rs *RelabelerStats) Add(stat RelabelerStats) {
+	rs.SamplesAdded += stat.SamplesAdded
+	rs.SeriesAdded += stat.SeriesAdded
+	rs.SeriesDrop += stat.SeriesDrop
+}
+
+// Adds another slice stats.
+func (rs *RelabelerStats) Adds(stats []RelabelerStats) {
```

Comment:
Maybe
```suggestion
func (rs *RelabelerStats) Add(stats ...RelabelerStats) {
```
instead of two methods